.github/workflows/build.yml

```diff
@@ -103,6 +103,25 @@ jobs:
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.name }}
     steps:
+
+    # sccache's handling of the /fp:fast MSVC compiler option is broken, so use our fork with the fix.
+    # https://github.com/mozilla/sccache/issues/950
+    - name: "[Windows] Set up cargo cache"
+      if: runner.os == 'Windows'
+      uses: actions/cache@v2
+      id: sccache-build-cache
+      with:
+        path: C:\Users\runneradmin\.cargo
```

Comment:
%USERPROFILE% = C:\Users\runneradmin?

Not much information only found this source: https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/environment-variables-full-list-github-actions

Reply:
As noted above, GH Actions caching doesn't seem work with Windows environment variables in the path.

Reply:
I will not bother experimenting with that further. It works how it is and the absolute path is unlikely to ever change.